src/main/webapp/js/email-browser.js

```diff
@@ -19,6 +19,74 @@
 var jasig = jasig || {};
 
 (function($, fluid) {
+
+	var ischecked = 0;
```

Comment:
@ChristianMurphy,

> is  `isChecked` counting the number of checks?

Yes, because I need to catch the global checkness state of every checkboxes in the Messages List. To handle every cases in Tablet's / Desktop's View :

- when  a user selects one or many checkboxes -> `.toolbar-middle` slides down and the user can suppress or not the selected message(s) or can selected all the displayed messages by selecting checkbox of `.toolbar-middle` ;

- when  a user has selected one or many checkboxes and he clicks on a message to display it -> because there is a suppress button on the displayed message, I flush `ischecked` to 0 to slide up `.toolbar-middle...`

So I've got the same wondering as you when I've created 'isChecked', but because I'm not a native english speaker (:tada: :fr:  :wine_glass: :confetti_ball:) , at this time :sweat_smile:, I did not know how to name simply `the counting of the checkboxesGlobalChecknessState`... If you've got a idea of something meanfull for us, please help me. :simple_smile: